src/game/common/audio/audioeventrts.cpp

```diff
@@ -311,6 +311,18 @@ void AudioEventRTS::Generate_Filename()
     }
 }
 
+/**
+ * Apply the filename inside the info directly
+ */
+void AudioEventRTS::Apply_Filename_From_Info()
```

Comment:
I added this function so i could pass a filename directly into an event - without any INI prefixes / suffixes

Reply:
It is an odd function, because the file name needs to be set with 2 function calls effectively. Perhaps make this a simple `Set_Filename(const Utf8String &name)` instead.

Also, it better be noted that this is a Thyme specific function to avoid confusion.